.eslintrc.cjs

```diff
@@ -9,30 +9,7 @@ module.exports = {
   // WARNING: If you want to add new rules/plugins, you need to add them to the eslint-plugin-cli package.
   extends: ['plugin:@shopify/eslint-plugin-cli/config'],
   rules: {
-    '@nx/enforce-module-boundaries': [
-      'error',
-      {
-        allow: [],
-        depConstraints: [
-          {
-            sourceTag: 'scope:feature',
-            onlyDependOnLibsWithTargs: ['scope:foundation'],
```

Comment:
These never actually worked -- onlyDependOnLibsWithTargs should be onlyDependOnLibsWithTags. Given they don't work and so aren't used, have switched to plain `error` instead.